zebra-script/Cargo.toml

```
@@ -8,7 +8,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-zcash_script = "0.1.6-alpha.0"
+zcash_script = { git = "https://github.com/jvff/zcash_script.git", rev = "2e4cd75" }
```

Security/Reliability: Let's use a full commit hash to the Foundation repository.

(GitHub's security model around forks is a bit weird. It might be possible for someone to mine another revision that starts with `2e4cd75` in their fork, and have it picked up in the main repository.)

```suggestion
zcash_script = { git = "https://github.com/ZcashFoundation/zcash_script.git", rev = "2e4cd75b0fa794af4eb0bd447ed14f0b1167d75a" }
```

---

Done in [2c3cba7](https://github.com/ZcashFoundation/zebra/commit/2c3cba7395a42b5fe96930188fcaf7e48d541520).